### PR TITLE
[MRG+1] Fix interactive topomap projections.

### PR DESCRIPTION
**Pull request by jaeilepp**
Closes https://github.com/mne-tools/mne-python/issues/4054

### CODE REVIEW

**Comment by agramfort** [comment]
@jasmainak can you confirm this fixes your pb

**Comment by jasmainak** [comment]
yes I can confirm that it fixes my problem.

File: mne/viz/topomap.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def _plot_update_evoked_topomap(params, bools):
     new_evoked.add_proj(projs)
     new_evoked.apply_proj()
 
-    data = new_evoked.data[np.ix_(params['picks'],
+    data = new_evoked.data[np.ix_(range(len(new_evoked.data)),
                                   params['time_idx'])] * params['scale']
```
Comment (Eric89GXL) [inline mne/viz/topomap.py]: At this point, can't you just do `new_evoked.data[:, params['time_idx']]`?

**Comment by jasmainak** [comment]
One thing I noticed by the way, is that some of the projs don't work. It's because you are plotting only magnetometers but the projs for the other channel types are also shown. It might be surprising for some users ...

**Comment by agramfort** [comment]
travis is not happy

**Comment by Eric89GXL** [comment]
It's an unrelated OSX download failure

**Comment by Eric89GXL** [comment]
LGTM +1 for merge

**Pull request merged**